apps/speed.c

```diff
@@ -932,7 +932,7 @@ static int EVP_Update_loop_aead_enc(void *args)
         /* Set length of iv (Doesn't apply to SIV mode) */
         if (mode_op != EVP_CIPH_SIV_MODE) {
             if (!EVP_CIPHER_CTX_ctrl(ctx, EVP_CTRL_AEAD_SET_IVLEN,
-                                     aead_ivlen, NULL)) {
+                                     AEAD_IVLEN, NULL)) {
```

Comment:
why not use `sizeof(aead_iv)` (here and at line 1007)? Then it doesn't matter if further changes are made to the declaration of `aead_iv`

Reply:
Agreed. Thanks @tom-cosgrove-arm. Will do another commit

Reply:
Done :)